src/main/java/com/synopsys/integration/blackduck/codelocation/signaturescanner/command/ToolsApiScannerInstaller.java

```diff
@@ -220,20 +217,6 @@ protected String downloadSignatureScanner(File scannerExpansionDirectory, HttpUr
         }
     }
 
-    /**
-     * Deletes legacy version file that may be left behind if client had previously installed scan-cli using {@link ZipApiScannerInstaller}
-     * @param scannerExpansionDirectory
-     */
-    private void removeOldVersionFileIfExists(File scannerExpansionDirectory) {
-        try {
-            File oldVersionFile = new File(scannerExpansionDirectory, ZipApiScannerInstaller.VERSION_FILENAME);
-            oldVersionFile.delete();
-        } catch (Exception e) {
-            logger.trace("Could not delete old Signature Scanner version file.");
-        }
-    }
-
-
```

Comment:
Instead of adding yet another "deleteNewVersionFileIfExists()" method ... we just go back to using blackDuckVersion.txt (old version filename) and not worry about which API we used. We update the same version file with each download.